e2e-tests/uat.env

```
@@ -62,4 +62,12 @@ CARD_TEST_DATA='
         }
     ]
 }'
-
+VALID_CARD_DATA='
+{
+    "testingPsp": "Wordline",
```

```suggestion
    "testingPsp": "Worldpay",
```

typo: this psp abi is worldpay, not worldline :D

---

Solved in https://github.com/pagopa/pagopa-checkout-tests/pull/112/commits/b850dec38eb7bb49063f848cab46e3b3a9789f17